src/d3a/models/myco_matcher/myco_matcher_interface.py

```diff
@@ -0,0 +1,41 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from typing import Dict
+
+
+class MycoMatcherInterface:
```

Comment:
Any reason why this is not an ABC class?

Reply:
Not really, It's just that I did not want to force the implementation of the methods defined here, but I can mark the class as abstract

Reply:
OK, no. I see now that this is more like a fallback and the single point of implementation for the `update_area_uuid_markets_mapping` that is used in both matcher cases. Right?
Then leave it as is. Thanks!